modules/claims_api/app/workers/claims_api/ews_updater.rb

```diff
@@ -16,11 +16,17 @@ def perform(ews_id)
       ews = ClaimsApi::EvidenceWaiverSubmission.find(ews_id)
       bgs_claim = benefit_claim_service(ews).find_bnft_claim(claim_id: ews.claim_id)
 
-      unless bgs_claim&.dig(:bnft_claim_dto, :filed5103_waiver_ind) == FILE_5103
+      indicator = bgs_claim&.dig(:bnft_claim_dto, :filed5103_waiver_ind)
+      if indicator.present? && indicator != FILE_5103
         bgs_claim[:bnft_claim_dto][:filed5103_waiver_ind] = FILE_5103
-
         update_bgs_claim(ews, bgs_claim)
       end
+      if bgs_claim&.dig(:bnft_claim_dto).blank?
+        ews.status = ClaimsApi::EvidenceWaiverSubmission::ERRORED
```

Comment:
this ERRORED status will get overwrote by the error from the `updte_claim_level_suspense`, and if not, it gets overwrote by the update on 31... I think this branch needs updated now that I'm looking at it, because that update for claim level suspense should be nested now ?